lib/CL/devices/cpu_dbk/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+
+if(HAVE_LIBJPEG_TURBO)
```

Comment:
@franz when you have a chance (not urgent), can you add turbojpeg to one of the CIs so we can test this automatically?